build_tools/ci/cpu_comparison/run.py

```diff
@@ -1180,8 +1221,8 @@ def all_tests(
     for test in tests.tests:
 
         # Determine if the test is a match for the test_set provided by caller
-        match = "All" in test_set
-        match = match or test.name in test_set
+        # match = "All" in test_set
```

Comment:
@yzhang93 before there was a special case check for "All" here (which I meant to completely remove) but this PR removes the special case and makes it just another label, but one which all tests have. 

Maybe it was simpler before this change